### ISSUE TITLE
cmd + s doesn't save scene

### ISSUE THREAD

**Issue by jasondkiesling** [isenseDev/MYR#87]

**Comment by theyoyojo** [comment]
Is this a MacOS only issue? I have no problem saving via ctrl-S on RHEL 7.6

```
[jsavitz@jsavitz ~]$ uname -a
Linux jsavitz.bos.csb 3.10.0-957.1.3.el7.x86_64 #1 SMP Thu Nov 15 17:36:42 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by theyoyojo** [comment]
Also, was this fixed in PR #112?

**Comment by jasondkiesling** [comment]
The issue is for cmd+s. There is no standard for the keycode for command which causes issues. A fix was attempted but more work is required. ctrl+s works.
